frontend/console/src/features/modules/decls/verb/VerbFormInput.tsx

```diff
@@ -18,13 +19,29 @@ export const VerbFormInput = ({
   onSubmit: (path: string) => void
   handleCopyButton?: () => void
 }) => {
+  const formRef = useRef<HTMLFormElement>(null)
+
   const handleSubmit: React.FormEventHandler<HTMLFormElement> = async (event) => {
     event.preventDefault()
     onSubmit(path)
   }
 
+  useEffect(() => {
+    const handleGlobalKeyDown = (event: KeyboardEvent) => {
+      if (event.altKey && event.key === 'Enter') {
+        event.preventDefault()
+        formRef.current?.dispatchEvent(new Event('submit', { cancelable: true, bubbles: true }))
+      }
+    }
+
+    document.addEventListener('keydown', handleGlobalKeyDown)
+    return () => {
+      document.removeEventListener('keydown', handleGlobalKeyDown)
+    }
+  }, [path, readOnly, onSubmit])
```

Comment:
Not super critical here, but we have a similar example of this in the searchInput for command pallete: https://github.com/TBD54566975/ftl/blob/main/frontend/console/src/layout/navigation/SearchInput.tsx#L10-L25

Most of this is basically the same, but you could add support for other `meta` keys as well like this:

```ts
  const shortcutText = window.navigator.userAgent.includes('Mac') ? '⌘ + K' : 'Ctrl + K'
```

Reply:
Yeah. considered the component and just avoided until a second case case popped up - but I had forgotten about the search input. What I'll do is align to the search, and come back and figure out a more generic component later.

Reply:
for sure! That's mostly what I meant too. No need to make something generic for just these 2 cases, but the command palette might be a good place to copy pasta from.